src/speech.cs

```diff
@@ -347,4 +359,55 @@ interface SFVoiceAnalytics : NSCopying, NSSecureCoding {
 		[Export ("voicing", ArgumentSemantic.Copy)]
 		SFAcousticFeature Voicing { get; }
 	}
+
+	[Mac(14, 0), iOS(17, 0), MacCatalyst(17, 0)]
+	[BaseType(typeof(NSObject))]
+	interface SFSpeechLanguageModelConfiguration : NSCopying {
+		[Export("initWithLanguageModel:")]
+		NativeHandle Constructor(NSUrl languageModel);
+
+		[Export("initWithLanguageModel:vocabulary:")]
+		NativeHandle Constructor(NSUrl languageModel, [NullAllowed] NSUrl vocabulary);
+
+		[Export("languageModel", ArgumentSemantic.Copy)]
+		NSUrl LanguageModel { get; }
+
+		[NullAllowed, Export("vocabulary", ArgumentSemantic.Copy)]
+		NSUrl Vocabulary { get; }
+	}
+
+	[Mac (14,0), iOS (17,0), MacCatalyst(17, 0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface SFSpeechLanguageModel {
+		[Static]
+		[Export ("prepareCustomLanguageModelForUrl:clientIdentifier:configuration:completion:")]
+		[Async]
+		void PrepareCustomModel (NSUrl asset, string clientIdentifier, SFSpeechLanguageModelConfiguration configuration, Action<NSError> completion);
+
+		[Static]
+		[Export ("prepareCustomLanguageModelForUrl:clientIdentifier:configuration:ignoresCache:completion:")]
+		[Async]
+		void PrepareCustomModel (NSUrl asset, string clientIdentifier, SFSpeechLanguageModelConfiguration configuration, bool ignoresCache, Action<NSError> completion);
+	}
+
+	[Partial]
+	[Mac (14,0), iOS (17,0), MacCatalyst(17, 0)]
+	interface SFAnalysisContextTag {
+		[Field("SFAnalysisContextTagLeftContext")]
+		NSString LeftContext { get; }
+
+		[Field("SFAnalysisContextTagRightContext")]
+		NSString RightContext { get; }
+
+		[Field("SFAnalysisContextTagSelectedText")]
+		NSString SelectedText { get; }
+	}
+
+	[Partial]
+	[Mac(14, 0), iOS(17, 0), MacCatalyst(17,0)]
+	interface SFSpeech {
+		[Field("SFSpeechErrorDomain")]
+		NSString ErrorDomain { get; }
+	}
```

Comment:
Remove this and use the [ErrorDomain ("SFSpeechErrorDomain")] attribute on top of `SFSpeechErrorCode`
```suggestion
```